### PR TITLE
chore: starting adding modules

### PR DESCRIPTION
**Pull request by edgesoft**

### CODE REVIEW

File: content/tutorial_next_step/02_adding_modules.en.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+})
+
+export default controller
+```
+
+What we have done now is add a namespace to our application called **app** and we reference our App module. We also have two other modules, **devtools** and **http**. That means there is no difference creating an application specific module or a module you want to share with others. Let us take a closer look at our **App** module:
+
+[src/modules/App/index.js](https://github.com/cerebral/cerebral-website-tutorial-next/blob/react/src/modules/App/index.js)
+
+```javascript
```
Comment (Guria) [inline content/tutorial_next_step/02_adding_modules.en.md]: is it from https://github.com/cerebral/cerebral-website-tutorial-next/blob/react/src/modules/App/index.js ?
don't inline it manually.
I will do it in build step in future.

File: content/tutorial-introduction/04_creating_components.en.md [hunk 2]
```diff
@@ -52,4 +52,4 @@ Make your components as dumb as possible. They should only define state dependen
 #### Challenge
 Create a new component called **Items** and move the UI description into that, also moving the state dependency path for the items. Make sure you import the component into App and make it a child.
 
-[Next](../get-started/get-started.en.md)
+([Next](../tutorial_next_step/01_next_step.en.md))
```
Comment (Guria) [inline content/tutorial-introduction/04_creating_components.en.md]: remove extra brackets please :)
it breaks my parser :(

Comment (edgesoft) [inline content/tutorial-introduction/04_creating_components.en.md]: Yes, will also remove the code

**Comment by edgesoft** [comment]
@Guria Ok, removed code and extra parentheses

**Comment by Guria** [comment]
:+1: also add new section to side-nav meta in root index.en.md as well.
keep adding rest of files to this folder.
 npm start right now fail because of broken next link to unexisting page. I'll fix it

also we must decide _ vs - in folder and file names and keep it consistent

**Comment by edgesoft** [comment]
@Guria Added the rest of the next step

**Comment by edgesoft** [comment]
@Guria Yes, right now we have _ in folder and file names but - in md files.

**Comment by edgesoft** [comment]
@Guria Ok, did as you suggested

**Pull request merged**